pytorch_blade/src/common_utils/BUILD

```diff
@@ -1,27 +1,54 @@
+package(default_visibility = ["//visibility:public"])
 load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
 
-filegroup(
-    name = "torch_blade_common_srcs",
-    srcs = glob(["**/*.cpp"], exclude = ["**/*_test.cpp"]),
+cc_library(
+    name = "torch_blade_logging",
+    hdrs = ["logging.h"],
+    deps = [
+       "@local_org_torch//:libtorch",
+       "@org_tensorflow//tensorflow/core/platform:logging",]
```

Comment:
Intend to use TF logging facilities? PyTorch-Blade depends on TF? Sounds weird.

Reply:
Yes. I think sources dependencies are OK since we had depends on BladeDISC.

Reply:
I think a good point is that this doesn't add new repository dependencies to TorchBlade. And the dependencies is narrow since it related to target "@org_tensorflow//tensorflow/core/platform:logging".